troubleshoot/ingest/opentelemetry/edot-collector/enable-debug-logging.md

```diff
@@ -0,0 +1,63 @@
+---
+navigation_title: Enable debug logging
+description: Learn how to enable debug logging for the EDOT Collector in supported environments.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_collector: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-collector
+---
+
+# Enable debug logging
+
+You can enable debug logging in the Elastic Distributions of OpenTelemetry (EDOT) Collector by setting the `--log-level=debug` flag. This is useful when troubleshooting startup issues or configuration problems.
+
+This guide shows how to enable debug logging in different environments.
+
+## Standalone EDOT Collector
+
+If you're running the EDOT Collector directly, add the flag to your command:
+
+```bash
+edot-collector --config=/path/to/otel-collector-config.yaml --log-level=debug
```

Comment:
Dumb question. So this is `edot-collector` because I'm running that collector directly. L53 mentions the "EDOT collector" but then uses `otel-collector` in the example. Is that correct?

Reply:
There's some switching between EDOT collector and OTel collector in this doc which is confusing to me. But maybe a more experienced user will understand the differences.

Reply:
That's kind of OK, but better if we can make it either more agnostic or more edot-specific, like `edot-collector-config.yaml`. In any case, if a default config file exists, let's go with it.